src/tests/config/karma.conf.common.js

```diff
@@ -52,9 +52,57 @@ export default function karmaConfigGenerator(webpackConfig, browserstackLauncher
         'karma-mocha',
         'karma-mocha-reporter',
         'karma-chai',
-        'karma-chai-spies'
+        'karma-chai-spies',
+        'karma-viewport'
       ],
 
+      viewport: {
```

Comment:
As we're no longer using `karma-viewport` we should remove the dependency, and all references in this file.